pkg/agent/main.go

```diff
@@ -147,33 +144,15 @@ func main() {
 	err = lightHouseAgent.Start(ctx.Done())
 	exitOnError(err, "Failed to start lighthouse agent")
 
-	httpServer := startHTTPServer()
+	defer http.StartServer(http.Metrics|http.Profile, 8082)()
```

Comment:
Previously it just started the metrics handler (L170).

Reply:
Yes, I forgot to mention that in the commit message — the goal of all this is to enable the profiling endpoints everywhere.

Reply:
So it should be 

```suggestion
	defer http.StartServer(http.Metrics|http.All, 8082)()
```

Reply:
That’s exactly why I didn’t use `All` here. I’ll change the Admiral PR to make `all` private.